docs/modules/ROOT/pages/explanations/decisions/user-management.adoc

```diff
@@ -0,0 +1,200 @@
+= Simple User management
```

Comment:
Is it really just "User Management", or is it in more general something like "Multi-Tenant Access Control" or "Shared Service Access Management"?

Reply:
I feel like the topic "Multi-Tenant Cluster Access" is a completely different beast, for example for something like Minio or Kafka, where there is a single instance for the whole cluster. That's why I'm calling this decision "Simple User management".

Currently, the user needs for PostgreSQL are: "I have a bunch of apps that need separate users".